lib/mumuki/domain/extensions/time.rb

```diff
@@ -0,0 +1,5 @@
+class Time
+  def round_years_since(another_time)
+    (self.to_s(:number).to_i - another_time.to_s(:number).to_i) / 10e9.to_i
```

Comment:
Why not `another_time.year - self.year`?

Reply:
I'm using this one to get user age. 
We need to handle month and day as well.

Reply:
Further thoughts? Perhaps the name is not ideal, but I couldn't come up with a better one and I decided to move this logic here instead of having it in the user class

Reply:
@flbulgarelli I'm missing this one to integrate gamification into master :smile:

Reply:
Ok. Go ahead.

However I think it can be more clear  and avoid that dangling `to_s` by using something like this, which is more intention revealing: 

```
(self.to_date -  another.to_date / 365).to_i
```

Reply:
In the end we decided to leave the code as-is, since  - although it is hacky - it allows to properly deal with leap years.